hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieDeleteBlock.java

```diff
@@ -97,6 +97,7 @@ public DeleteRecord[] getRecordsToDelete() {
     }
   }
 
+  // TODO(HUDI-5760) avoid using Kryo for serialization here
```

Comment:
Is this comment misplaced? I am assuming this is to avoid Kryo for serialization to disk. So, it should be placed before `getContentBytes`?

Reply:
Good point. Duplicated this comment in both ser/de paths